### DISCUSSION TITLE
note: This error originates from a subprocess, and is likely not a problem with pip.

### DISCUSSION THREAD

**Discussion by FangA186** [explosion/spaCy#13371] [Help: Other Questions] [answered]
![image](https://github.com/explosion/spaCy/assets/119376573/8dc9033f-c607-4305-a444-8dff4ca67585)

**Comment by FangA186** [comment]
![image](https://github.com/explosion/spaCy/assets/119376573/5e95b13a-bbbd-4719-a98b-83234e6281d0)
pip install chatterbot error

**Comment by svlandeg** [comment, marked as answer]
Hi! Sorry to hear you're running into trouble.

[Chatterbot](https://github.com/gunthercox/ChatterBot) is a plugin that isn't maintained by us though - so best to ask your question on their repo.
